packages/appstore/src/components/cfds-listing/index.tsx

```diff
@@ -172,6 +172,16 @@ const CFDsListing = observer(() => {
         return null;
     };
 
+    useEffect(() => {
+        if (is_landing_company_loaded && is_populating_mt5_account_list) {
```

Comment:
@mitra-deriv why are you waiting for just mt5? What about cTrader and DXtrade?

Reply:
@sergei-deriv I added a console log the whole cfd section loads by these conditions and at same time I'm setting the performance value I assume we don't need additional checks here , if you think still we need those checks let me know thanks